SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -112,6 +113,8 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	UPROPERTY()
 	USpatialReceiver* Receiver;
 	UPROPERTY()
+	UActorGroupManager* ActorGroupManager;
```

Comment:
Seems like the only usage of `ActorGroupManager` in `SpatialNetDriver` is creation and being passed to `ClassInfoManager`.

Is there any reason the `ActorGroupManager` shouldn't just be init'd by and existing solely in the `ClassInfoManager`?